src/common/pool_map.c

```diff
@@ -490,7 +490,7 @@ pool_buf_parse(struct pool_buf *buf, struct pool_domain **tree_pp)
 	/* Initialize the root */
 	parent = &tree[0]; /* root */
 	parent->do_comp.co_type   = PO_COMP_TP_ROOT;
-	parent->do_comp.co_status = PO_COMP_ST_UP;
+	parent->do_comp.co_status = PO_COMP_ST_UPIN;
```

Comment:
It's not from this patch, but I'm wondering why don't we just use the state returned from server directly instead of setting one arbitrarily?

Reply:
Yes, getting from pool buf might make more sense.   And probably you can change target status to UPIN in init_pool_metadata(). Of course once we totally support integrate process. The initial status should be UP, then change to UPIN once reintegrate finish.

Reply:
Because this is the root, perhaps? I guess @gnailzenh wrote this line of code.

Reply:
Yes this is the root, which should always be UPIN.